e2e/src/helper.ts

```diff
@@ -58,11 +58,18 @@ export async function activateAndWait(docUri: Uri): Promise<void> {
 
 function onDidChangeTextDocument(e: TextDocumentChangeEvent): void {
   console.log(JSON.stringify(e.contentChanges));
-  if (e.contentChanges.length === 1 && e.contentChanges[0].text === '') {
-    return;
-  }
 
   if (e.document.uri.path === 'Preview' && previewPromiseResolve) {
+    if (
+      // When we switch to a new document, the preview content is set to '' we skip this such events here
+      e.contentChanges.length === 1 &&
+      e.contentChanges[0].text === '' &&
```

Comment:
Please check, if it should be ' ' rather than '' ?

Reply:
It should be ''. We receive such events when we loose focus from sql file for example and preview is cleared.